lib/perl/Genome/Config/AnalysisProject/Command/Release.pm

```diff
@@ -46,15 +46,27 @@ sub execute {
 sub __errors__ {
     my $self = shift;
     my @errors = $self->SUPER::__errors__(@_);
-    for my $status (map{$_->status} $self->analysis_projects){
+
+    for my $anp ($self->analysis_projects) {
+        my $status = $anp->status;
         unless(grep{$_ eq $status} ("Pending", "Hold", "In Progress")){
             push @errors, UR::Object::Tag->create(
                 type => 'error',
                 properties => ['analysis_projects'],
-                desc => "Can't release analysis project with status: $status"
+                desc => "Can't release analysis project " . $anp->__display_name__ . " with status: $status"
+            );
+        }
+
+        my $config = $anp->environment_config_dir;
+        unless ($config) {
```

Comment:
The [`environment_config_dir`](https://github.com/tmooney/genome/blob/63fb747959461b19b699507aefc568bfdf8dcea3/lib/perl/Genome/Config/AnalysisProject.pm#L151) method does validate the path exists before returning it, but it doesn't check that there's really anything specified.

Would it be beneficial to add some extra checking there?  I considered parsing the environment in advance to look for particular keys, but it's not guaranteed that we'll need to use them for a particular build type.  We could at least look for some valid configuration file at all, though?